src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+const handleSuccess = (message) => {
+  const successDiv = document.createElement('div');
+
+  successDiv.classList.add('message');
+  successDiv.textContent = message;
+  document.body.appendChild(successDiv);
+};
+
+const handleError = (errorMessage) => {
+  const errorDiv = document.createElement('div');
+
+  errorDiv.classList.add('message', 'error-message');
+  errorDiv.textContent = errorMessage;
+  document.body.appendChild(errorDiv);
+};
```

Comment:
Same as for the 'handleSuccess' function, the 'handleError' function could lead to performance issues if called multiple times. Consider updating the content of an existing div instead of creating a new one each time.